modules/videoio/test/test_camera.cpp

```diff
@@ -104,5 +104,42 @@ TEST(DISABLED_videoio_camera, v4l_read_framesize)
 
     capture.release();
 }
+TEST(COUNT_OF_CAMERA_FRAMES, v4l_poll)
+{
+        VideoCapture cap1(0);
+        VideoCapture cap2(2);
+
+        std::vector<VideoCapture> VCM;
+
+        VCM.push_back(cap1);
+        VCM.push_back(cap2);
+
+        std::vector<int> State;
+
+        std::vector<int> CountOfStates(2, 0);
+
+        int iteration = 500;
```

Comment:
Numeric constants like this can be declared at the beginning of the test:
```.cpp
const unsigned ITERATION_COUNT = 500;
const unsigned TIMEOUT = 50;
```